### PR TITLE
docs(configuration): improve `devServer` documentation

### PR DESCRIPTION
**Pull request by Biki-das**
Improved the overall document fixing wordings adding necessary punctuations and making it more easier to read.

### CODE REVIEW

**Comment by Biki-das** [comment]
@snitin315  hope you have a look on this!

File: src/content/configuration/dev-server.mdx [hunk 1]
```diff
@@ -71,9 +72,9 @@ W> Be aware that when [exporting multiple configurations](/configuration/configu
 
 T> If you're having trouble, navigating to the `/webpack-dev-server` route will show where files are served. For example, `http://localhost:9000/webpack-dev-server`.
 
-T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on your configuration, URL may look like `http://localhost:9000/invalidate`.
+T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on your configuration,the URL may look like `http://localhost:9000/invalidate`.
```
Comment (snitin315) [inline src/content/configuration/dev-server.mdx:75]:
```suggestion
T> If you want to manually recompile the bundle, navigating to the `/invalidate` route will [invalidate](/api/node/#invalidate-watching) the current compilation of the bundle and recompile it for you via [webpack-dev-middleware](https://github.com/webpack/webpack-dev-middleware#invalidate). Depending on your configuration, the URL may look like `http://localhost:9000/invalidate`.
```

File: src/content/configuration/dev-server.mdx [hunk 2]
```diff
@@ -693,7 +694,7 @@ To disable:
 npx webpack serve --no-history-api-fallback
 ```
 
-By passing an object this behavior can be controlled further using options like `rewrites`:
+Bypassing an object this behavior can be controlled further using options like `rewrites`:
```
Comment (snitin315) [inline src/content/configuration/dev-server.mdx:697]:
```suggestion
By providing an object this behavior can be controlled further using options like `rewrites`:
```

File: src/content/configuration/dev-server.mdx [hunk 3]
```diff
@@ -964,7 +965,7 @@ module.exports = {
 
 `function (devServer)`
 
-Provides the ability to execute custom middleware prior to all other middleware
+Provides the ability to execute custom middleware before all other middleware
```
Comment (snitin315) [inline src/content/configuration/dev-server.mdx:968]: What's wrong with `prior to`?

File: src/content/configuration/dev-server.mdx [hunk 4]
```diff
@@ -1120,7 +1121,7 @@ module.exports = {
 };
 ```
 
-T> The browser application name is platform dependent. Don't hard code it in reusable modules. For example, `'Chrome'` is `'Google Chrome'` on macOS, `'google-chrome'` on Linux and `'chrome'` on Windows.
+T> The browser application name is platform-dependent. Don't hard code it in reusable modules. For example, `'Chrome'` is `'Google Chrome'` on macOS, `'google-chrome'` on Linux , and `'chrome'` on Windows.
```
Comment (snitin315) [inline src/content/configuration/dev-server.mdx:1124]:
```suggestion
T> The browser application name is platform-dependent. Don't hard code it in reusable modules. For example, `'Chrome'` is `'Google Chrome'` on macOS, `'google-chrome'` on Linux, and `'chrome'` on Windows.
```

File: src/content/configuration/dev-server.mdx [hunk 5]
```diff
@@ -1332,7 +1333,7 @@ module.exports = {
 
 `boolean` `string` `[string]` `object` `[object]`
 
-This options allows to configure options for serving static files from directory (by default 'public' directory). To disable set it to `false`:
+This option allows configuring options for serving static files from the directory (by default 'public' directory). To disable set it to `false`:
```
Comment (snitin315) [inline src/content/configuration/dev-server.mdx:1336]: What's wrong with `allows to configure`?

**Review by snitin315 — COMMENTED** [review]
Thanks for the PR, I have few comments.

File: src/content/configuration/dev-server.mdx (hunk 3 above)
Comment (Biki-das) [reply src/content/configuration/dev-server.mdx:968]: Prior to has similar meaning to before ,you can check below.

prior to - Google Search - Google Chrome 26-09-2021 8 59 08 AM

File: src/content/configuration/dev-server.mdx (hunk 5 above)
Comment (Biki-das) [reply src/content/configuration/dev-server.mdx:1336]: Nothing wrong with allows to configure, this whole sentence is in present continuos tense and allows configuring seems to fit in here that why i did that

**Comment by Biki-das** [comment]
thanks for the feedback!

**Comment by Biki-das** [comment]
> thanks for the feedback! @snitin315  and made the changes as you suggested

File: src/content/configuration/dev-server.mdx (hunk 3 above)
Comment (snitin315) [reply src/content/configuration/dev-server.mdx:968]: Then let's stick with `prior to`.

Comment (Biki-das) [reply src/content/configuration/dev-server.mdx:968]: > Then let's stick with `prior to`.

so can you suggest me the changes or i should make a new Pullrequest ?

Comment (snitin315) [reply src/content/configuration/dev-server.mdx:968]:
```suggestion
Provides the ability to execute custom middleware prior to all other middleware
```

File: src/content/configuration/dev-server.mdx [hunk 6]
```diff
@@ -51,7 +52,7 @@ module.exports = {
 };
 ```
 
-When the server is started, there will be a message prior to the list of resolved modules:
+When the server is started, there will be a message before the list of resolved modules:
```
Comment (snitin315) [inline src/content/configuration/dev-server.mdx:55]:
```suggestion
When the server is started, there will be a message prior to the list of resolved modules:
```

**Review by snitin315 — APPROVED** [review]
LGTM. Thanks!

**Comment by Biki-das** [comment]
> LGTM. Thanks!

Happy to help always! :-)

**Pull request merged**